frontend/public/components/chargeback.tsx

```diff
@@ -174,42 +173,46 @@ const DataCell = ({name, value}) => {
   return value;
 };
 
-const DataTable = ({rows, orderBy, sortBy, applySort, keys}:DataTableProps) => {
+const DataTable = ({cols, rows}:DataTableProps) => {
   const size = _.clamp(Math.floor(12 / _.size(rows[0])), 1, 4);
   const className = `col-md-${size}`;
-  return <div className="co-m-table-grid co-m-table-grid--bordered" style={{marginTop: 20, marginLeft: -15, marginRight: -15}}>
-    <ListHeader>
-      {_.map(keys, k => <ColHead
-        className={classNames(className, {'text-right': _.isFinite(_.get(rows, [0, k]))})}
-        key={k}
-        sortField={k}
-        sortFunc={k}
-        currentSortOrder={orderBy}
-        currentSortField={sortBy}
-        currentSortFunc={sortBy}
-        applySort={applySort}>
-        {k.replace(/_/g, ' ')}
-      </ColHead>
-      )}
-    </ListHeader>
-    <div className="co-m-table-grid__body">
-      {_.map(rows, (r, i) => <div className="row co-resource-list__item" key={i}>
-        {_.map(r, (v, k) => <div className={className} key={k}><DataCell name={k} value={v} /></div>)}
-      </div>)}
-    </div>
-  </div>;
+
+  const DataTableHeader = () => _.map(cols, col => {
+    const value = _.find(rows[0], (rowValue, key) => {
+      return key === col;
+    });
+    return ({
+      sortFunc: _.isFinite(value) ? 'chargebackData' : null,
```

Comment:
I have a similar concern here. You're just checking a single value to see if it looks like a number, but you don't know if all values in that column are numbers. We might wait until we convert to the v2 API when we know the actual type.

Reply:
Hi @spadgett, reverted the sorting changes and believe I have addressed latest review comments.  PTAL  -thanks